roles/ansible-tdp-common-actions/tasks/config-chrony-for-vms.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Comment default chronyd makestep
+  replace:
+    dest: /etc/chrony.conf
+    regexp: '^makestep'
+    replace: '#makestep'
+
+- name: Set chronyd makestep
+  blockinfile:
+    path: /etc/chrony.conf
+    backup: yes
+    content: |
+      # Sync 1+ second clock deviations at any time
+      makestep 1 -1
```

Comment:
From a first look, it doesn't seem idempotent. Here is my interpretation of the code's workflow:

First run

1. Comment `makestep`
2. Add a new line `makestep 1 -1`

Second run:

3. Comment line created in step 2
4. Add a new line `makestep 1 -1`

Instead, I will suggest to merge the first 2 steps with:

```
- name: Aggressive time sync with 1+ second clock deviations at any time for dev purpose
  replace:
    dest: /etc/chrony.conf
    regexp: '^makestep.*$'
    replace: 'makestep 1 -1'
 ```

Reply:
@wdavidw You're right. It's been fixed in the last commit. I added file backup on `chrony.conf` as there is no longer the default value in the file.